book/src/collabSNARKs-primer.md

```diff
@@ -1,3 +1,33 @@
 # Collaborative SNARKs Primer
 
-Collaborative SNARKs (coSNARKs) are a very new line of work, [emerging in 2021](https://eprint.iacr.org/2021/1530.pdf), that combines the best of MPC and SNARKs. They enable multiple parties to jointly create a proof of the correctness of a computation while keeping their individual inputs private. This approach solves the problem of computing on private shared state without relying on a trusted third party. Each party contributes to a distributed protocol that generates a single SNARK proof, attesting to the correctness of their combined computation without revealing any individual inputs. The resulting proof is succinct and can be verified efficiently by anyone, ensuring the integrity of the computation without exposing any sensitive data.
+**Collaborative SNARKs (coSNARKs)** are a recent innovation, [introducing in 2021](https://eprint.iacr.org/2021/1530.pdf), that merge the strengths of MPC and zkSNARKs. They allow multiple parties to collaboratively create a proof verifying the correctness of a computation while keeping their individual inputs private. This method addresses the challenge of computing on private shared data without needing a trusted third party. Each party participates in a distributed protocol that produces a single SNARK proof, confirming the accuracy of the combined computation without revealing any individual inputs. The resulting proof is succinct and can be efficiently verified by anyone, ensuring the integrity of the computation while safeguarding sensitive data.
+
+To highlight what we can achieve with coSNARKs imagine two parties, Alice and Bob. 
+
+<img src="figures/step1.png" alt="alice-bob" width="100%"/>
+
+Alice and Bob own a secret value $a$ and $b$ respectively. They want to compute the [Poseidon](https://eprint.iacr.org/2019/458.pdf) hash of their combined secrets, but without revealing $a$ or $b$ to each other.
```

Comment:
maybe bring the proof aspect of this in here, in that they not only compute the hash, but also compute a zksnark of its validity